packages/stream-metadata/src/routes/spaceRefresh.ts

```diff
@@ -14,7 +15,7 @@ const paramsSchema = z.object({
 		}),
 })
 
-// This route handler validates the refresh request and quickly returns a 200 response.
+// This route handler validates the refresh request, initiates the CloudFront invalidation, and returns a 200 response.
```

Comment:
@miguel-nascimento i like what you're doing here. good job!

question: does the client need to be aware of the invalidation id? if not, i suppose you're using this to communicate the id between the route and the onResponse hook. this is not a big deal but using the context pattern might be better here

Reply:
The client needs the `invalidationId` so it can get the `refreshStatus` later with the same id.

I'm passing in the headers just to communicate between route <> onResponse, it was the easiest way (I think the correct way here would be `AsyncLocalStorage`)